.circleci/config.yml

```diff
@@ -133,6 +139,31 @@ jobs:
             ./cc-test-reporter upload-coverage
           when: on_success
 
+  e2e_file_upload_features:
+    <<: *job_defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: /home/circleci/
+      - run:
+          <<: *install_dependencies
+      - run:
+          name: RUN_E2E_TESTS_FILE_UPLOAD_FEATURES
```

Comment:
Sorry, should have told you this earlier, but can you change the name so that it is `Run e2e file upload features test`

Reply:
Done.